micromark/exports-html.mjs

```diff
@@ -1,7 +1,5 @@
 // @ts-check
 
-/* eslint-disable n/no-missing-import */
```

Comment:
Please undo both instances of this edit. The disable is necessary in the general case and only unnecessary if someone runs `npm install` in the `micromark` directory. Removing like this will fail CI.

Reply:
Yes. Or clean the micro ark directory and it should go away.